ecosystem/typescript/sdk/src/tests/e2e/indexer.test.ts

```diff
@@ -359,7 +359,16 @@ describe("Indexer", () => {
       "gets account coin data",
       async () => {
         const accountCoinData = await indexerClient.getAccountCoinsData(alice.address().hex());
-        expect(accountCoinData.current_coin_balances[0].coin_type).toEqual("0x1::aptos_coin::AptosCoin");
+        expect(accountCoinData.current_fungible_asset_balances[0].asset_type).toEqual("0x1::aptos_coin::AptosCoin");
```

Comment:
Do we cover both standards  (`coin.move` and `fungible_asset.move`) when writing these unit tests? Otherwise we can miss some stuff like with the thing I posted a couple days ago where it works as expected for one but borks the other

I can help you write some of them, if we don't have them